broadinstitute/orsp-pub
# BTRX - 616 - Convert Fundings Report

## Addresses
[BTRX-616](https://broadinstitute.atlassian.net/browse/BTRX-616) : ORSP: Convert Funding Source Report

## Changes
- Implement a common bootstrap table 2 component
- Use this common component in Funding Report and Comments
- Remove Fundings report related grails files

## Testing
- From top nav options, navigate to Fundings Source Report. The table should be looking good. Funding reports is a paginated table, so pagination should also be working good.
- Export to excel should be working.
- Export to CSV should be working
- Print also should be working

---
